metaflow/plugins/kfp/tests/sample_flows/nested_foreach_with_branching.py

```diff
@@ -0,0 +1,114 @@
+from metaflow import FlowSpec, step
+
+
+def truncate(var):
+    var = str(var)
+    if len(var) > 500:
+        var = "%s..." % var[:500]
+    return var
```

Comment:
is this still necessary?  It was remarked upon earlier from other files and removed.